detections/endpoint/windows_set_public_network_profile_via_registry.yml

```
@@ -1,11 +1,11 @@
-name: Windows Set Private Network Profile via Registry
+name: Windows Set Public Network Profile via Registry
```

A better title would be

```suggestion
name: Windows Set Network Profile Category to Private via Registry
```